recipes/esys-escript/meta.yaml

```diff
@@ -20,7 +20,7 @@ requirements:
     build:
         - {{ compiler('cxx') }}
         - svn
-        - scons >3.0.5
+        - scons >3.0.5 py27_0
         - cmake
         - make
         - python 2.7.*
```

Comment:
@aellery, this is enough to use python 2 scons. You can remove the `py27_0` above

Reply:
I have done this.